projects/1.20/assets/dungeons-delight/dungeons_delight_gourmet_battles/lang/zh_cn.json

```
@@ -0,0 +1,91 @@
+{
+  "advancements.crispyandsweet.descr": "这些腿尝起来怎么样？",
```

```suggestion
  "advancements.crispyandsweet.descr": "这些小细腿尝起来怎么样？",
```
漏译，原文`How do those little legs taste between your teeth?`